public/src/components/ToastManager.jsx

```diff
@@ -24,35 +26,37 @@ const ToastRack = styled.div`
   z-index: 1;
   position: fixed;
   right: 0;
-  width: 250px;
+  ${({ theme }) => css`
+    width: ${theme.toast.width};
+  `}
   bottom: 0;
   padding: 1em;
 `;
 
 const StyledToastBanner = styled.div`
   margin-left: -1px;
-    border-radius: .25em 0 0 .25em;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    width: 25px;
-    & path {
-      fill: white;
-    }
-    padding: .25em .5em;
-    ${(props) => getCSSForFlavor(props.flavor)}
+  border-radius: .25em 0 0 .25em;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 25px;
+  & path {
+    fill: white;
+  }
+  padding: .25em .5em;
+  ${(props) => getCSSForFlavor(props.flavor)}
 `;
 const StyledToastContent = styled.div`
   padding: .5em 1em;
-    flex: 1;
+  flex: 1;
 `;
 const StyledToastCancel = styled.div`
   display: flex;
-    padding: .25em .5em;
-    & path {
-      fill: #54457F;
-    }
-    align-items: center;
+  padding: .25em .5em;
+  & path {
```

Comment:
`& selector` isn't necessary, but I get if it's there for stylistic purposes

`selector` is equivalent to `& selector`

Reply:
Gotcha, I have no strong preference either way, I just figure `& selector` was clear.